app/models/user.py

```diff
@@ -17,6 +21,40 @@ class User(baseModel):
 
     def __str__(self):
         return str(self.__dict__)
+    
+    def __init__(self,*args, **kwargs):
+        super().__init__(*args,**kwargs)
+        self._ldsCache = None
+    
+    @property
+    def isLaborDepartmentStudent(self):
+        if self._ldsCache is not None:
+            return self._ldsCache
+       
+        if not self.student:
+            self._ldsCache = False
+            return False
+        
+        from app.models.laborStatusForm import LaborStatusForm
+        today = date.today()
+
+        labor_status_forms = (
+            LaborStatusForm
+            .select()
+            .join(Department, on=(LaborStatusForm.department == Department.departmentID))
+            .where(
+                (LaborStatusForm.studentSupervisee == self.student) &
+                (LaborStatusForm.startDate <= today) &
+                (LaborStatusForm.endDate >= today) &
+                (Department.isActive == True) &
+                (fn.BINARY(Department.DEPT_NAME) == "Labor Department") # comparison case sensetive.
+            )
```

Comment:
We need to make sure that students who are released from their position are excluded from this query. And also that this labor status form is approved. The first can be a subselect in the where clause, and the second a join with FormHistory (I think)